tests/stress_tests/.automation/memcheck_tests/nightly_configs/myriad_references_config.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<attributes>
+    <models>
+        <!--Models with FP32 precision-->
+        <model path="public/alexnet/FP32/alexnet.xml" precision="FP32" test="create_exenetwork" device="MYRIAD" vmsize="2219614" vmpeak="2807734" vmrss="522615" vmhwm="959597" />
```

Comment:
please, add comment with which OpenVINO version references were collected (like `# values from {"commit_id": "761e571042fa2b291d5954e523fffc1e2dfcafae", "commit_date": "2021-05-20 10:36"} and *= 1.3`)